src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -329,6 +330,7 @@ public enum FileAttribute
         Directory = 16,
         Archive = 32,
     }
+    [System.Diagnostics.CodeAnalysis.RequiresDynamicCodeAttribute("The FileSystem module is not supported in AOT environments. Use members of the System.IO namespace instead.")]
     [Microsoft.VisualBasic.CompilerServices.StandardModuleAttribute]
     public sealed partial class FileSystem
```

Comment:
Many but not all APIs of this class use `GetCallingAssembly` (some did call it only in debug asserts) but I didn't bother and marked the whole class. This class is very outdated either way.